README.md

```diff
@@ -53,8 +54,8 @@ In one sentence, explain why.
 
 ## A1.2
 
-«replace this with your answer»
-
+O(n), because n/2 of the elements of the path are pushed onto the stacked to be parsed 
+which is simplified to O(n).
```

Comment:
yup (although actually n+1 will eventually get pushed, because it doesn't know how many tiles there are until it gets to the end)